calico/maintenance/ebpf/enabling-bpf.md

```diff
@@ -81,10 +81,6 @@ eBPF mode has the following pre-requisites:
   needs a way to reach the API server directly.
 - The base [requirements]({{site.baseurl}}/getting-started/kubernetes/requirements) also apply.
 
-> **Note**: The default kernel used by EKS is not compatible with eBPF mode.  If you wish to try eBPF mode with EKS,
```

Comment:
Just to confirm, the current default kernel (if you create an EKS cluster with no particular options) is new enough for BPF?